evm/src/cpu/kernel/asm/core/call.asm

```diff
@@ -250,3 +257,78 @@ global after_call_instruction:
     %jump(memcpy)
 %%after:
 %endmacro
+
+// Charge gas for *call opcodes and return the sub-context gas limit.
+// Doesn't include memory expansion costs.
+%macro call_charge_gas
+    // Compute C_aaccess
+    // stack: cold_access, address, gas, kexit_info, value, args_offset, args_size, ret_offset, ret_size
```

Comment:
Am I reading it right that `value` is the deepest stack item this uses? If so I think we can remove the others - we usually only comment on args that a macro uses